src/test/java/com/slack/kaldb/testlib/KaldbConfigUtil.java

```diff
@@ -6,8 +6,8 @@
 
 public class KaldbConfigUtil {
   // Initialize kaldb config with empty for tests.
-  public static void initEmptyConfig() throws InvalidProtocolBufferException {
-    KaldbConfig.initFromJsonStr("{}");
+  public static void initEmptyIndexerConfig() throws InvalidProtocolBufferException {
```

Comment:
Since the config file also has a query service config, can we add the query service to the roles also? That way, the function name also doesn't need to change.

Reply:
which config file? Any reason you don't want to change the name of the function?